### DISCUSSION TITLE
POST messages to a web url  (webhook connection)

### DISCUSSION THREAD

**Discussion by SHAliakbari** [aahnik/tgcf#221] [Q&A] [answered]
Is it possible to send received message to a web url instead of forwarding messages ?! 

I mean, in some situations my app needs to get the messages, right now i forward message to another group and get the forwarded message with my bot and process it. It will be good if tgcf send received message directly to my app.

**Comment by aahnik** [comment, marked as answer]
You want to setup a webhook connection. That's an awesome idea.

That is easily possible, by writing a tgcf plugin. I will write an official plugin for this soon. Watch the repo to stay updated.

To know more about plugins read the wiki.
